common/api/core-bentley.api.md

```diff
@@ -222,6 +222,12 @@ export enum ChangeSetStatus {
     WrongDgnDb = 90130
 }
 
+// @internal
+export namespace ClassUtils {
```

Comment:
Did you consider what if any value the namespace provides?

Reply:
I'm not sure what you mean by this, can you clarify?

Reply:
`const isSubClass = ClassUtils.isSubClassOf<T>(blah);`
vs
`const isSubClass = isSubClassOf<T>(blah);`
It's a nitpick, this isn't Java, functions don't need to live inside otherwise-meaningless classes/namespaces.

Reply:
you're right I think it would be better to give consumers the choice to export it by name or do a whole namespace import, I'll change this

Reply:
done